client/src/components/scorecard_page/ScorecardMapViewBar.vue

```diff
@@ -115,8 +117,13 @@ export default defineComponent({
   background-color: $warm-grey-100;
 }
 
+@media screen and (max-width: 470px) {
+  .container {
+    justify-content: center;
```

Comment:
No way to do this in Bulma

Reply:
does https://bulma.io/documentation/helpers/typography-helpers/#alignment help?